src/fetch/PageAction.js

```diff
@@ -0,0 +1,113 @@
+import { pageAction } from "./prompts.js";
+import { logger } from "../log/logger.js";
+import { TagRemovingMinimizer } from "../min/TagRemovingMinimizer.js";
+import { getAI } from '../ai/index.js';
+import { Timer } from "../log/timer.js";
+
+export const PageAction = class {
+  constructor(namespace, prompt) {
+    this.namespace = namespace;
```

Comment:
actually maybe we don't need this and we can keep track of categories of these somewhere else